pkg/controller/plan/adapter/vsphere/client.go

```diff
@@ -150,10 +160,30 @@ func (r *Client) PowerOff(vmRef ref.Ref) (err error) {
 	if powerState == types.VirtualMachinePowerStatePoweredOff {
 		return nil
 	}
+
+	// Check if this is Windows Server 2019
+	// Windows Server 2019 may enter hibernation instead of fully powering off when PowerOff() is called,
+	// causing virt-v2v to fail.
+	isWindows2019 := isWindows(vmModel) && strings.Contains(vmModel.GuestID, "2019")
+
 	err = vm.ShutdownGuest(context.TODO())
 	if err != nil {
-		err = liberr.Wrap(err)
-		return
+		if isWindows2019 {
+			// For Windows Server 2019, don't attempt hard power off - return the ShutdownGuest error
+			r.Log.Info("ShutdownGuest failed for Windows Server 2019, not attempting hard power off", "vmRef", vmRef, "error", err.Error())
+			err = liberr.Wrap(err, "ShutdownGuest failed for Windows Server 2019")
+			return
+		}
+
+		r.Log.Info("ShutdownGuest failed, attempting hard power off", "vmRef", vmRef, "error", err.Error())
+		// Fallback to asynchronous hard power off when VMware Tools aren't available
+		_, powerOffErr := vm.PowerOff(context.TODO())
```

Comment:
Ah, this is hard power off, didn't realise it thought it was still graceful.
In that case, let's fail the migration, so we can just update the message or have this as known issue.
The hard power off can cause disk corruption or lost data don't want to go to that direction.